sway-lib-std/src/alloc.sw

```diff
@@ -45,3 +46,12 @@ pub fn realloc(ptr: u64, size: u64, new_size: u64) -> u64 {
         ptr
     }
 }
+
+// Allocate a type on the stack.
+pub fn alloca<T>() -> u64 {
+    let current_pointer = stack_ptr();
+    asm() {
+        cfei i32;
```

Comment:
This is allocating 32 bytes, where it needs to allocate '__size_of<T>()`.

Reply:
> This is allocating 32 bytes, where it needs to allocate '__size_of()`.

Oops, yes. Let me fix this.

Reply:
Dropping (and reverting related commits) this as it needs to be an intrinsic. The argument to `cfei` must be an immediate :-(